test/pageobjects/wikiPage.js

```diff
@@ -0,0 +1,48 @@
+class WikiPage {
+
+    get searchInput() {
+        return $('//*[@id="searchInput"]')
+    }
+
+    get searchButton() {
+        return $('//*[@id="search-form"]/fieldset/button/i')
+    }
+
+    async searchForArticle(articleName) {
+        await this.searchInput.setValue(articleName);
+        await this.searchButton.click();
+    }
+
+    get toolsButton() {
+        return $('//*[@id="vector-page-tools-dropdown-checkbox"]')
+    }
+
+    get downloadPdfLink() {
+        return $('//*[@id="coll-download-as-rl"]/a/span');
+    }
+
+    async navigateToDownloadPDF() {
+        await this.toolsButton.click();
+        await this.downloadPdfLink.scrollIntoView();
+        await this.downloadPdfLink.click();
+    }
+
+    get pdfFileName() { 
+        return $('//*[@id="mw-content-text"]/form/div/label/div[2]/div[2]').getText(); 
```

Comment:
Absolute path, we don't  use it, you need to change this locator

Reply:
Resolved